### DISCUSSION TITLE
Ui version

### DISCUSSION THREAD

**Discussion by notaf0e** [fungamer2-2/Life-Simulator1#59] [Ideas]
I was thinking of making bitlife in python too, but with a ui using Kivy. Im very new to Kivy but have some experience with python. Could i use the code from this project in one of my own?: BitPyfe https://github.com/NotAF0e/BitPyfe

**Comment by fungamer2-2** [comment]
Sure, that sounds like a good idea

**Reply by notaf0e** [reply]
Cool i will clone the project into my project as it is MIT license, then there will be a link to this project

**Reply by notaf0e** [reply]
Actually I changed my mind. I've just looked at your code and realized how big of an undertaking this will be. I'm a student studying for GCSE so this would have been too much for me

**Reply by fungamer2-2** [reply]
> Actually I changed my mind. I've just looked at your code and realized how big of an undertaking this will be. I'm a student studying for GCSE so this would have been too much for me

Oh ok
